docs/index.rst

```diff
@@ -26,6 +26,12 @@ Features
 * Queries ResultsDB for automated test results and displays them on updates.
 
 
+System Requirements
+-------------------
+
+Bodhi is currently supported on all active Fedora releases.
```

Comment:
+ only?

Reply:
I changed it to "Bodhi is currently only supported on active Fedora releases."